src/components/AuthRoute.js

```diff
@@ -41,13 +41,17 @@ type AuthRouteState = {
 // LJV TODO Create a no-permissions page to use for unauthorized users
 class AuthRoute extends React.Component<AuthRouteDefaultProps, AuthRouteProps, AuthRouteState> {
   static defaultProps = {
-    required: null,
+    requiredRole: null,
     location: null, // This should be filled in by the router
     authType: 'NONE',
   };
 
   static displayName = 'AuthRoute';
 
+  static childContextTypes = {
+    auth: PropTypes.any,
```

Comment:
Do you want to give children access to the AuthRoute component? Or just the user info object? It might make more sense to include just the user info in the context...

However, in case you really do need the whole AuthRoute component, BTW, if you look at ListSerarchResult, you'll see an example of a better PropTypes usage here... You can do:
```auth: PropTypes.instanceOf(AuthRoute);```
I should really update the rest of the occurrences in the code to do the same since the use of `any` is discouraged and in newer versions of Flow will be a warning or error...
But I digress....

Reply:
Just the user info,
That makes sense, I'll update it.